app.json

```diff
@@ -17,5 +17,5 @@
   "name": "dvc.org",
   "scripts": {
   },
-  "stack": "heroku-16"
+  "stack": "heroku-18"
```

Comment:
This is so automatic review apps (https://dashboard.heroku.com/pipelines/7a105706-a0cc-4e66-a022-2ea44c7cf238) match the main app's updated Heroku Stack version.